### DISCUSSION TITLE
Detailed ranking data

### DISCUSSION THREAD

**Discussion by flodorner** [LAION-AI/Open-Assistant#3394] [Q&A]
If I understand correctly, the oasst1 dataset only contains ranking data that has been aggregated by "Tideman’s method" (according to the paper). As there are other aggregation schemes and little research on their different merits in conjunction with RLHF, are there plans to include more detailed ranking information (i.e. counts for each unique personal ranking) in future dataset releases?

**Comment by olliestanley** [comment]
Currently this hasn't been planned, but it would theoretically be possible (same for labels rather than simply releasing aggregates). Some work would be required to support it in the export script, which someone would need to volunteer to take on
